tests/phpunit/tests/Block/Web_Stories_Block.php

```diff
@@ -26,16 +26,17 @@
  */
 class Web_Stories_Block extends Test_Case {
 	public function tearDown() {
-		unregister_block_type( \Google\Web_Stories\Block\Web_Stories_Block::BLOCK_NAME );
+		unregister_block_type( 'web-stories/embed' );
 
 		parent::tearDown();
 	}
 
 	/**
 	 * @covers ::register
+	 * @covers ::register_block_type
 	 */
 	public function test_registers_block_type() {
-		$this->assertTrue( WP_Block_Type_Registry::get_instance()->is_registered( \Google\Web_Stories\Block\Web_Stories_Block::BLOCK_NAME ) );
+		$this->assertTrue( WP_Block_Type_Registry::get_instance()->is_registered( 'web-stories/embed' ) );
```

Comment:
The test is failing because `assets/js/block.json` does not exist during test run, so there's no block to be registered.

Relates to my concern about the location of `block.json` in https://github.com/google/web-stories-wp/pull/7970#issuecomment-867800535

Reply:
Yeah, having to build the js for PHPUnit tests feels super wasteful. I think this workaround is fine.